### DISCUSSION TITLE
About SofaAPAPI software installation

### DISCUSSION THREAD

**Discussion by andHeath** [InfinyTech3D/SofaUnity#73] [Help - Using SofaUnity]
Hi,
I want to cut a cube using a knife and create a 3D shape as a result.
The cutting process takes place, but the cutting occurs randomly. It does not cut the point where the blade touches the earring.
I need assistance

**Comment by epernod** [comment]
Hi,

could you show which scene you are using and also where is the cutting plan defined in your scene?

Best regards

**Comment by epernod** [comment]
Hi,

as I didn't get any response from you, I'm closing this discussion thread. Feel free to reopen it or ask other questions in another thread.

Best regards,
